source/slang/slang-ast-base.h

```diff
@@ -405,20 +489,22 @@ class Decl : public DeclBase
 
     ContainerDecl* parentDecl = nullptr;
 
+    DeclRefBase* defaultDeclRef = nullptr;
```

Comment:
Could use a comment indicating what this is/means.

Reply:
Commented.